internal/helm/chartyaml_test.go

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package helm
+
+import "testing"
+
+func TestShouldPassIfCanReadChartYaml(t *testing.T) {
+	c := chartYamlFromFile("test/TestChartValid.yaml")
+	if c.Name != "test-application" {
+		t.Errorf("Test should pass but name of the helm chart is not 'test-application'.")
+	}
+}
+
+func TestShouldPassIfChartVersionIsValid(t *testing.T) {
```

Comment:
I would prefer a better name, that documents the business rule. 
The current name is too technical in my option (exaggerated ShouldBeValidIfValid -> yes of course :))
Proposal: `TestShouldBeValidForSemanticVersion`

Reply:
Function renamed to TestShouldBeValidForSemanticVersion